jeta/base.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC
+
+from jax import random
+
+
+class BaseLearner(ABC):
+    """Base Learner class
+
+    Attributes:
```

Comment:
```suggestion

    Args:
```

In other docstrings, we are using `Args` only.   So to be consistent with the documentation, I would suggest using "Args" only.

Reply:
Or if this is intentional or any docstring format is decided, then let me know.